### ISSUE TITLE
Can't pay Lightning invoice - Says lightning invoice is invalid

### ISSUE THREAD

**Issue by gblilley** [LN-Zap/zap-desktop#3717]
## Description
I have two open channels and both are online. Both channels have a "can send capacity" greater than the amount of the invoice. From the wallet, I press "Pay" and then paste a valid lightning invoice into the payment request box, and I get an immediate message right below the box that says "Not a valid bitcoin/lightning request". I have tried pasting invoices created by different lightning wallets and none of them are considered valid.

## Expected Behavior
To pay the invoice out of my available funds.

## Actual Behavior
Get a message: "Not a valid bitcoin/lightning request"

## Steps to Reproduce
- 1. Press "Pay" Button from the wallet.
- 2. Paste a lightning invoice into the Payment Request box.
- 3. Get a message: "Not a valid bitcoin/lightning request"

## Your Environment
Laptop Windows 10 - 64 bit
Zap Version  0.7.2

**Comment by bolatovumar** [comment]
@gblilley if you paste an invoice here (https://lightningdecoder.com) which Zap tells you is invalid, what do you get? Does it work?

**Comment by gblilley** [comment]
That's a handy tool. Thanks. With your help, I figured out the problem.

The Lightning invoice that is generated is preceded by "lightning:". The decoder treats it as a valid address if you include the "lightning:" prefix when you paste the address, and it also treats it as valid if you omit the "lightning:" prefix. 

When I paste the address into Zap, it treats the address as invalid if you include the "lightning:" prefix, but once you remove it, Zap processes the invoice correctly.  

Thanks very much for your help. Until I started experimenting with the decoder, I had no idea what was going on.
